searchobject/src/main/java/org/jalau/at18/searchobject/middleware/FaceDetectionControllerMiddleware.java

```diff
@@ -94,18 +96,23 @@ public void doFilter(ServletRequest request,
                 pw.flush(); //delete the content of the txt file.
                 pw.close();
                 fw.close();
-                throw new MiddlewareException("    Token has no more uses, please request another one");
+                throw new MiddlewareException(" Token has no more uses, please request another one");
             } else {
                 LOG.warning(" GENERATED TOKEN ");
-                throw new MiddlewareException("    Generated token ");
+                throw new MiddlewareException("Generated token ");
             }
         } catch (InstantiationError | MiddlewareException e) {
             LOG.warning(" ERROR LOADING THE MODEL " + e);
             e.printStackTrace();
-            PrintWriter out = response.getWriter();
             res.setStatus(400);
-            out.println ("    Status :    " + res.getStatus());
-            out.println (e.getMessage());
+            String x = Integer.toString( res.getStatus());
+            ErrorResponse employee = new ErrorResponse(x, e.getMessage());
```

Comment:
Please rename this variable `employee` to a name more descriptive according the project

Reply:
variable name changed